services/airflow/dags/update_hospitalizations/transform_dataframe.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+from isoweek import Week
+from datetime import datetime
+from numpy import datetime64 as d64
+from numpy import timedelta64 as td64
+
+
+def transform_dataframe(df):
+    population = 83020000  # TODO replace hardcoded population with reference that will be updated
```

Comment:
You can get the latest population for germany by querying the postgres db for 
```postgresql
SELECT 
    number 
FROM 
    censusdata.population 
WHERE 
    nuts='DE' AND 
    agegroup='TOTAL' AND 
    sex='T' 
ORDER BY 
    year DESC 
LIMIT 1;
```

Reply:
Just make sure to run the `load_nuts_regions` and `load_population_for_nuts_regions` tasks in your tests before to populate the database.